lib/dai-plugin-mcd/test/Auction.spec.js

```diff
@@ -0,0 +1,40 @@
+import Maker from '@makerdao/dai';
+import { mcdMaker } from './helpers';
+const { DAI, ETH, MKR } = Maker;
+
+describe('flip auction', () => {
+  sharedTests(ETH);
```

Comment:
there's a `describe.each` syntax built into Jest that you can use in this case (the Cdp.spec case is more complicated): https://jestjs.io/docs/en/api.html#describeeachtable-name-fn-timeout

Reply:
Good to know, thanks.  Do you think the CdpTypeService test should iterate through all the collateral types?  Right now I'm just testing the ETH collateral type

Reply:
Btw, I think this PR is ready to be merged in now.